src/scss/components/checkbox/mixins/_hxCheckboxControl.scss

```diff
@@ -0,0 +1,112 @@
+@import "vars";
+@import "hxCheckboxFacade";
+
+//
+// ===== Mixins for <hx-checkbox-control> states =====
+//
+
+// ========== PRISTINE ==========
+@mixin __checkboxControl($psuedo-state: null) {
```

Comment:
It's spelled `pseudo` (`e` before `u`), not `psuedo`. 😄

Reply:
😆 thanks for the catch!